starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/annotationProcessor.rocker.raw

```diff
@@ -6,7 +6,7 @@ String artifact
 )
 
 @if (features.language().isJava()) {
-@if (features.testFramework().isKotlinTest()) {
+@if (features.testFramework().isKotlinTest() || features.testFramework().isKoTest()) {
```

Comment:
This could be simpler if this PR was merged: https://github.com/micronaut-projects/micronaut-starter/pull/411